.editorconfig

```diff
@@ -5,3 +5,7 @@ trim_trailing_whitespace = true
 
 [Makefile]
 trim_trailing_whitespace = true
+
+[src/hotspot/**.{cpp,hpp,h}]
+indent_style = space
```

Comment:
Just wondering - will the new src/hotspot block inherit the trim_trailing_whitespace setting from the top-level [*] block?

Reply:
From documentation that I checked it should, but I'll test it just to be sure